packages/@react-types/dialog/src/index.d.ts

```diff
@@ -36,10 +36,20 @@ export interface SpectrumBaseDialogProps extends HTMLAttributes<HTMLElement> {
 }
 
 export interface SpectrumDialogProps extends DOMProps, StyleProps {
+  /** The contents as semantic elements to display in the dialog. */
   children: ReactNode,
+  /**
+   * [Slots](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/slot)
+   * defining semantic names for dialog children.
```

Comment:
This link describes a `<slot>` element which is something different from what `slots` is for Dialog. Maybe something like "Object containing the props that each Dialog child receives. Organized by `slot`, see [LINK TO SLOT DOCUMENTATION]."?